integration-tests/application-server-integration-tests/src/test/resources/Dockerfile-wildfly-latest

```diff
@@ -0,0 +1,4 @@
+FROM quay.io/wildfly/wildfly:27.0.0.Final-jdk17
```

Comment:
[For Reviewer]
This is intentionally not the very latest version, which is `28`. This is to check whether the dependabot approach works as desired.